src/components/clusterConfiguration/ClusterConfiguration.tsx

```diff
@@ -173,6 +172,10 @@ const ClusterConfiguration: React.FC<ClusterConfigurationProps> = ({ cluster })
 
     // update the cluster validation
     try {
+      if (!values.pullSecret) {
```

Comment:
This will get removed once the pull secret is not available from API, right? If so, a TODO would be nice

Reply:
This will remain here since the `pullSecret` will remain among `cluster-update-params`. With this patch, it is not read from `Cluster` anymore, but remains among formik's fields since to support the "update" flow. The `delete` here ensures that the backend will not update its value when not intended to.